Description: Fix FTBFS on mips64el.
 The header which defines 'struct xdr_ops' has no mention of these
 special IRIX fields so when they're inserted later in 'xdrposix_ops'
 they cause all the other fields to be misplaced and the incorrect
 functions get called.
Author: James Cowgill <james410@cowgill.org.uk>
Bug-Debian: https://bugs.debian.org/779624

--- a/mfhdf/libsrc/xdrposix.c
+++ b/mfhdf/libsrc/xdrposix.c
@@ -291,12 +291,6 @@ static void xdrposix_destroy();
 static struct xdr_ops   xdrposix_ops = {
     xdrposix_getlong,   /* deserialize a 32-bit int */
     xdrposix_putlong,   /* serialize a 32-bit int */
-#if (_MIPS_SZLONG == 64)
-    /* IRIX64 has 64 bits long and 32 bits int. */
-    /* It defines two extra entries for get/put int. */
-    xdrposix_getint,   /* deserialize a 32-bit int */
-    xdrposix_putint,   /* serialize a 32-bit int */
-#endif
     xdrposix_getbytes,  /* deserialize counted bytes */
     xdrposix_putbytes,  /* serialize counted bytes */
     xdrposix_getpos,    /* get offset in the stream */
